test/e2e/_helpers.js

```diff
@@ -5,21 +5,46 @@ import { join } from 'path'
 import { format } from 'date-fns'
 import faker from 'faker'
 import glob from 'glob'
-import { find, isArray, isNil } from 'lodash'
-import { ClientFunction, t } from 'testcafe'
+import { find, get, isArray, isNil } from 'lodash'
+import { ClientFunction, RequestLogger, t } from 'testcafe'
 
+import moveService from '../../common/services/move'
 import personService from '../../common/services/person'
+import referenceDataService from '../../common/services/reference-data'
+import { formatDate } from '../../config/nunjucks/filters'
+
+let referenceData
+let genders
+let ethnicities
+let locations
+
+const getReferenceData = async () => {
+  genders = await referenceDataService.getGenders()
```

Comment:
These could be done using `Promise.all` to speed things up a little if needed. They don't really need to be run synchronously in this order.

Reply:
true dat.

Reply:
Except we've now moved to calling the ref data service directly so this has gone away